### PR TITLE
[calculator] setting up React and MicroFrontend for calculator project

### PR DESCRIPTION
**Pull request by 98H**

### CODE REVIEW

**Review by ainyava — APPROVED** [review]
Great, everything works, tested :+1:

File: apps/calculator/src/index.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import React from 'react';
```
Comment (ketabi) [inline apps/calculator/src/index.js:1]: You could import it from shared remote (zarkit)

File: apps/calculator/webpack.config.js [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+  output: {
+    publicPath: 'http://localhost:3003/',
+  },
+  resolve: {
+    extensions: [
+      ".jsx",
+      ".js",
+      ".tsx",
+      ".ts",
```
Comment (ketabi) [inline apps/calculator/webpack.config.js:23]: We don't need it

File: apps/calculator/webpack.config.js [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+      ".jpg",
+      "jpeg",
+      "png",
+    ],
+  },
+  module: {
+    rules: [
+      {
+        test: /\.(jpg|png|gif|jpeg)$/,
+        loader: "url-loader",
```
Comment (ketabi) [inline apps/calculator/webpack.config.js:36]: You can check it PR for @ainyava

**Review by ketabi — APPROVED** [review]
Add your imports for calculator in Host app.
You could use bootstrap.js from @behradhajmiri and setup standalone project.
- index.js
- ...
-

**Comment by ketabi** [comment]
@98H 
Why not merged?

**Pull request merged**